rocketpy/environment/environment_analysis.py

```diff
@@ -8,6 +8,11 @@
 import numpy as np
 import pytz
 
+try:
+    from functools import cached_property
+except ImportError:
+    from ..tools import cached_property
```

Comment:
You can actually deleted the try-except statement and simply import using `from functools import cached_property`. This will work fine in any environment with python>3.8, so we are covered.

Also, there are a few more files where we do this kind of try-except.

![image](https://github.com/RocketPy-Team/RocketPy/assets/63590233/b36748dd-5a87-46ce-8748-f64d7f0c8a14)